spec/models/skill_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Skill, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
pending?

Reply:
This file is generated by default by rspec. If I leave block empty it will throw me remark that block can't be empty. Please advice what better to do in this case: leave it or delete skill_spec.rb and all other files generated by rspec with the same content from "rspec/models" directory?